great_expectations/rule_based_profiler/util.py

```diff
@@ -242,36 +242,80 @@ def compute_quantiles(
         metric_values,
         q=(false_positive_rate / 2),
         axis=0,
-        interpolation="linear",  # can be omitted ("linear" is default)
     )
     upper_quantile = np.quantile(
         metric_values,
         q=1.0 - (false_positive_rate / 2),
         axis=0,
-        interpolation="linear",  # can be omitted ("linear" is default)
     )
     return lower_quantile, upper_quantile
 
 
-def compute_bootstrap_quantiles(
+def _compute_bootstrap_quantiles_point_estimate_legacy(
     metric_values: np.ndarray,
     false_positive_rate: np.float64,
     n_resamples: int,
 ) -> Tuple[Number, Number]:
     """
-    Internal implementation of the "bootstrap" estimator method, returning confidence interval for a distribution.
-    See https://en.wikipedia.org/wiki/Bootstrapping_(statistics) for an introduction to "bootstrapping" in statistics.
+    An internal implementation of the "bootstrap" estimator method, returning a point estimate for a population
+    parameter of interest (lower and upper quantiles in this case). See
+    https://en.wikipedia.org/wiki/Bootstrapping_(statistics) for an introduction to "bootstrapping" in statistics.
+
+    This implementation has been replaced by "compute_bootstrap_quantiles_bias_corrected_point_estimate" and only
+    remains to demonstrate the performance improvement achieved by correcting for bias. Upon the implementation of a
+    Machine Learning Lifecycle framework, the performance improvement can be documented and this legacy method can be
+    removed from the codebase.
```

Comment:
Justification for leaving in the legacy method.

Reply:
Wow, really well-cited and explained. Thank very much @NathanFarmer